ext/dom/php_dom.stub.php

```diff
@@ -157,7 +157,7 @@ public function replaceChild(DOMNode $node, DOMNode $child) {}
 }
 
 /** @not-serializable */
-class DOMNameSpaceNode
+class DOMNamespaceNode
```

Comment:
I checked, and this has been called `DOMNameSpaceNode` in PHP 7 as well, so this was not an accidental rename during stubs migration (https://github.com/php/php-src/blob/23e13e2c8f1aa711d8129831c0e1f089cd2e85d1/ext/dom/php_dom.c#L662). Why are you renaming the class now?

Reply:
Yeah, I recently changed it back from `DOMNamespaceNode`, when I realized that I made the mistake back then.

Reply:
Ohh, I did not realize that it has been named like this from the start. It's weird, but if that's the case then I'll drop the commit. Luckily, class names are case-insensitive so this will not cause issues when people spell it correctly in their projects. It would be nice to follow some naming convention in PHP standard though. e.g. Pascal case